package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "agroeasy",
-  "version": "1.1.5",
+  "version": "1.1.6",
```

Comment:
how do we resolve version bumping issue, i increased to same version as you did.
@enyeInc your attention is needed here

Reply:
Think about it, whoever merges down their code first has the rightful claim to the version bump. The second person would then need to bump their version once more. I will not put anything in place to enforce it, if we agree to that.

Reply:
I think this is good for now.

Reply:
if eric pushed first, then he has the right to "version": "1.1.5" 
i will take "version": "1.1.6"
@Simpcyclassy you know what to do

Reply:
we have to avoid future version bump

Reply:
Yeah I totally agree